### DISCUSSION TITLE
Create dynamic response Headers (usefull with CRUD?)

### DISCUSSION THREAD

**Discussion by jpeter06** [mockoon/mockoon#1054] [Ideas]
Hello all, 

I am simulating"filtering" with CRUD routes
and found the need to return dynamic headers, in this case
to calculate number of responses and return a custom "**X-Total-Count**"
should be nice to be able to have a response helper like:
`{{ resp_header 'label' value }}`

really happy using mockoon,
nice work :-)

**Comment by 255kb** [comment]
Headers already [support templating](https://mockoon.com/docs/latest/templating/overview/#headers-templating). So, you can use the usual helpers in their values.
Then, for the filtering it would be a matter of having a `filter` helper (I put the issue on the roadmap). 

I am not very keen to add response headers, we already have [one](https://mockoon.com/docs/latest/templating/mockoon-response-helpers/) to set the HTTP status from the template but it doesn't feel that natural. however, for the status there wasn't any proper alternative.
